helm/harbor-operator-app/values.yaml

```diff
@@ -0,0 +1,247 @@
+image:
+  # image.repository -- The image repository whose default is the chart appVersion.
+  repository: goharbor/harbor-operator
```

Comment:
are you planning to add these to be retagged by retagger?

Reply:
Yep at some point... I just want to do some testing our clusters first